src/eggCookingHelper.tsx

```diff
@@ -0,0 +1,255 @@
+import { useEffect, useState } from "react";
+
+function createAudio() {
+    const audio = new Audio("/alert.mp3");
+    audio.loop = true;
+    return audio;
+}
+
+export default function EggCookingHelper() {
+    type EggSize = "S" | "M" | "L";
+    type BoilingLevel = "Soft" | "Medium" | "Hard";
+
+    const [eggSize, setEggSize] = useState<EggSize>("M");
+    const [boilingLevel, setBoilingLevel] = useState<BoilingLevel>("Medium");
+    const [remainingTimerSeconds, setRemainingTimerSeconds] = useState(0);
+    const [intervalID, setIntervalID] = useState<number>();
+    const [alertActive, setAlertActive] = useState(false);
+    const [audio] = useState(createAudio());
+
+    const cookingTimes = new Map<EggSize, Map<BoilingLevel, number>>();
+    const minutes = Math.floor(remainingTimerSeconds / 60);
+    const seconds = remainingTimerSeconds % 60;
+
+    cookingTimes.set(
+        "S",
+        new Map<BoilingLevel, number>()
+            .set("Soft", 3)
+            .set("Medium", 5)
+            .set("Hard", 7)
+    );
+
+    cookingTimes.set(
+        "M",
+        new Map<BoilingLevel, number>()
+            .set("Soft", 4)
+            .set("Medium", 6)
+            .set("Hard", 8)
+    );
+
+    cookingTimes.set(
+        "L",
+        new Map<BoilingLevel, number>()
+            .set("Soft", 5)
+            .set("Medium", 7)
+            .set("Hard", 9)
+    );
+
+    function calcBoilTime(eggSize: EggSize, boilingLevel: BoilingLevel) {
+        const sizeMap = cookingTimes.get(eggSize);
+        if (!sizeMap) {
+            return;
+        }
+        const cookingTime = sizeMap.get(boilingLevel);
+        if (!cookingTime) {
+            return;
+        }
+        const timeInSeconds = cookingTime * 60;
+        return timeInSeconds;
+    }
+
+    function reduceTimer() {
+        setRemainingTimerSeconds((remainingTimerSeconds) => {
+            if (remainingTimerSeconds > 1) {
+                return remainingTimerSeconds - 1;
+            }
+            setAlertActive(true);
+            setIntervalID((intervalID) => {
+                clearInterval(intervalID);
+                return undefined;
+            });
+            audio.play();
+            return 0;
+        });
+    }
+
+    useEffect(() => {
+        const boilingSeconds = calcBoilTime(eggSize, boilingLevel);
+        if (!boilingSeconds) {
+            return;
+        }
+        setRemainingTimerSeconds(boilingSeconds);
+    }, [eggSize, boilingLevel]);
+
+    return (
+        <div className="bg-[rgb(204,193,173)] flex justify-center overflow-hidden">
+            <div className="bg-[url('/kitchen-bg.png')] bg-no-repeat bg-contain bg-center border-x-8 border-[rgb(165,133,116)] h-screen aspect-[980/931]">
```

Comment:
Inspecting this element in the browser you can see an unwanted space on the top and bottom of that image. This is caused by a combination of the border and the aspect ratio. By default the element's total size is defined *including* its border. So the border should be respected in the element's aspect ratio.
Alternatively you could take a look at the `box-sizing` CSS prop which would be the cleaner way to solve this here.